lib/tools/adb-emu-commands.js

```diff
@@ -5,42 +5,46 @@ import path from 'path';
 
 let emuMethods = {};
 
-emuMethods.isEmulatorConnected = async function (udid) {
-  let emulators = await this.getConnectedEmulators();
-  if (!emulators.length) {
-    return false;
+emuMethods.getEmulators = async function() {
```

Comment:
an easier way to handle caching the value without needing to store state on the object would be to use `_.memoize`:

```js
emuMethods.getEmulators = _.memoize(async function() {
  return await this.getConnectedEmulators();
});
```